src/sdk/namespace_fs.js

```diff
@@ -162,6 +164,20 @@ class NamespaceFS {
             params.xattr_copy; // TODO, DO we need to hard link at TaggingDirective 'REPLACE'?
     }
 
+    run_update_issues_report(object_sdk, err) {
+        if (!err.code) return;
+        if (!this.namespace_resource_id) return;
```

Comment:
@jeniawhite - This is for standalone nsfs where we don't have a resource id, and do not even have the pool server to send the update_issues_report api call. So when namespace_resource_id is undefined, we skip reporting.

@liranmauda - You should definitely add comments to explain this in the constructor, and also here.

Reply:
This is per @guymguym request https://github.com/noobaa/noobaa-core/pull/6549#discussion_r646101060
In `nsfs.js` which is standalone, we want to avoid the report.

Reply:
Add a comment to explain that in the code!